base/src/main/proto/spine/options.proto

```diff
@@ -180,59 +180,49 @@ extend google.protobuf.FieldOptions {
     //
     // The value of the option is the name of the target event message/context field
     // which is used to set the corresponding enrichment's field.
-    // The value must have event/context FQN prefix if `enrichment_for` option is omitted
-    // ("example.events.MyEvent.user_id" instead of just "user_id").
+    // The value must have event/context plain field name (i.e. FQN names are not supported).
```

Comment:
A general suggestion: let's not use `FQN` abbreviation in the end user documentation. We can always say “a fully-qualified name”. 

Regarding this field documentation, the value of `(by)` option may contain more than one field reference, separated with the pipe character.